### PR TITLE
fix: init command not copied utils files

### LINKED ISSUE #75: `init` command is not generating all files
I tried running the following command:
```bash
bunx @apa-dev/ui@sigma init
```
However, it only generated the apa.config.json file, along with empty component and utils folders. The utils folder should contain the cn.ts file, but it is missing.

Environment:
- Next.js 15 project
- macOS
- Bun package manager

### LINKED ISSUE #77: `class-variance-authority` is not automatically installed on `init` command
```bash
https://nextjs.org/docs/messages/module-not-found
 ⨯ ./src/components/Button.tsx:5:1
Module not found: Can't resolve 'class-variance-authority'
  3 | import React from 'react'
  4 | import { cn } from '@/utils/cn'
> 5 | import { cva, type VariantProps } from 'class-variance-authority'
    | ^
  6 |
  7 | const variants = cva(
  8 |     'h-10 flex justify-center items-center rounded border-2 border-black bg-sky-300 text-sm font-semibold shadow-[3px_3px_0_0_#000] transition-all hover:translate-x-[3px] hover:translate-y-[3px] hover:shadow-none',

https://nextjs.org/docs/messages/module-not-found
```

### PR DESCRIPTION
**Pull request by powxenv**
resolves #75 
resolves #77

### CODE REVIEW

File: package-lock.json
Comment (zainphp) [inline package-lock.json:1]: lah, emang boleh `package-lock.json`❓❓❓❓

**Review by dominosaurs — CHANGES_REQUESTED** [review]
Also please fix the typo in:
https://github.com/apa-developer/apa-ui/blob/7d1791da26496995cc7abccaf9620915a8ee83b7/src/commands/init.ts#L129

File: package-lock.json
Comment (powxenv) [reply package-lock.json:1]: lah mana saya tau

**Comment by powxenv** [comment]
> Also please fix the typo in:
> 
> https://github.com/apa-developer/apa-ui/blob/7d1791da26496995cc7abccaf9620915a8ee83b7/src/commands/init.ts#L129

baik

**Review by dominosaurs — APPROVED** [review]
GELAS KINK 🍻

**Pull request merged**